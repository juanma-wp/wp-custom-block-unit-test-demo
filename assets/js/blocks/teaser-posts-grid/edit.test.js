import React from 'react';
import renderer from 'react-test-renderer';
import Edit from './edit';

const mockEditProps = {
	attributes: {
		items: [],
	},
	setAttributes: jest.fn(),
};

beforeEach( () => {
	mockEditProps.setAttributes.mockClear();
} );

describe( 'Teaser Posts Grid Block Edit Component', () => {
	test( 'snapshot renders with defaults', () => {
		const component = renderer.create( <Edit { ...mockEditProps } /> );
		expect( component.toJSON() ).toMatchSnapshot();
	} );
} );
