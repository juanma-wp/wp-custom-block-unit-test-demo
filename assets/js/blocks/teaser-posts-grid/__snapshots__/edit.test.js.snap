// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Teaser Posts Grid Block Edit Component snapshot renders with defaults 1`] = `
<div>
  <h2>
    Select some posts
  </h2>
  <ul />
  <button />
</div>
`;
